test/integration/tests.js

```diff
@@ -73,7 +73,7 @@ test('Gets an entry with a specific locale', (t) => {
     })
 })
 
-test('Get entry with fallback locale', (t) => {
+test.skip('Get entry with fallback locale', (t) => {
```

Comment:
What broke here? 😢

Reply:
Just the test space needs some special setup once, I should remove the skipping